docs/user/security/audit-logging.asciidoc

```diff
@@ -392,6 +392,10 @@ Refer to the corresponding {es} logs for potential write errors.
 | `success` | User has accessed a case comment as part of a search operation.
 | `failure` | User is not authorized to search for case comments.
 
+.2+| `case_categories_get`
+| `success` | User has accessed a case.
```

Comment:
Which docs are these? Should it be "User has accessed the categories of a case"?